taskcluster/ci/tests/kind.yml

```diff
@@ -0,0 +1,28 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+task-defaults:
+  description: "Run tests"
+  run:
+    using: run-task
+    cwd: '{checkout}'
+
+tasks:
+  run-my-test:
+    worker-type: b-linux-large
+    worker:
+      max-run-time: 86400
+      docker-image: {in-tree: testing}
+    run:
+      command:
+        - poetry install
```

Comment:
A list of commands is not supported (that would be nice.) But you can use && or run a bash script